### PR TITLE
Add mso_rest data source

### PR DESCRIPTION
**Pull request by danischm**

### CODE REVIEW

**Review by akinross — CHANGES_REQUESTED** [review]

**Review by akinross — APPROVED** [review]
LGTM

File: mso/datasource_mso_rest.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			"content": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		}),
+	}
+}
+
+func datasourceMSORestRead(d *schema.ResourceData, m interface{}) error {
+	log.Printf("[DEBUG] Beginning Read")
```
Comment (sajagana) [inline mso/datasource_mso_rest.go:33]:
```suggestion
	log.Printf("[DEBUG] %s: Beginning Read", d.Get("path").(string))
```

**Review by sajagana — CHANGES_REQUESTED** [review]

File: mso/datasource_mso_rest.go (hunk 1 above)
Comment (danischm) [reply mso/datasource_mso_rest.go]: Updated

**Review by sajagana — APPROVED** [review]
LGTM!

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**